### ISSUE TITLE
sh_8x8.py has a wrong function name

### ISSUE THREAD

**Issue by Teusia** [AOtools/soapy#118]
Hello, 
I am just getting started with soapy and to verify my installation I tried to run the example you give [here](https://github.com/AOtools/soapy). For the configuration file I tried using the [sh_8x8.py](https://github.com/AOtools/soapy/blob/master/conf/py_config/sh_8x8.py). 

However there is at least one function name that changed: on running `sim.aoinit()` python produced the following error message: 
```python
  File "C:\Users\anne.cheffot\anaconda3\envs\poppydev\lib\site-packages\soapy-unknown-py3.8.egg\soapy\wfs\shackhartmann.py", line 429, in calculateSlopes
    slopes = getattr(centroiders, self.config.centMethod)(

AttributeError: module 'aotools.image_processing.centroiders' has no attribute 'brightestPxl'
```
In centroiders.py I found a `brightest_pixel` but not a `brightestPxl`. After changing line 63 of `sh_8x8.py` from 
```python 
"centMethod"    :   ["brightestPxl"],
```
to 
```python
"centMethod"    :   ["brightest_pixel"],
```
`aoinit` went through.
Maybe you want to have a look at the soapy example files and make sure all function name have been updated to the latest version?
Cheers,
Anne-Laure Cheffot

**Comment by matthewtownson** [comment]
Yes, this appears to have not been updated when the function was renamed in aotoools. Thanks for pointing it out, we will sort it for the next release.

**Comment by Teusia** [comment]
:) thanks, Should I leave the issue open as a reminder?

**Comment by matthewtownson** [comment]
Yes, we should leave it open until it is fixed. Otherwise I am likely to forget.
